layers/+tags/gtags/README.org

```diff
@@ -5,121 +5,249 @@
 
 * Install
-** GNU Global
 
-You can install =helm-gtags= from the software repository of your OS. For example, in Ubuntu:
+** GNU Global (gtags)
 
-#+BEGIN_SRC sh
-  sudo apt-get install global
-#+END_SRC
+   To use =gtags=, you first have to install [[https://www.gnu.org/software/global/download.html][GNU Global]].
 
-To use =helm-gtags=, you first have to install [[https://www.gnu.org/software/global/download.html][GNU Global]].
+   You can install =global= from the software repository of your OS; however,
+   many OS distributions are out of date, and you will probably be missing
+   support for =pygments= and =exuberant ctags=, and thus support for many
+   languages. We recommend installing from source. If not for example to install
+   on Ubuntu:
 
-Download the latest tar.gz archive, then run these commands:
+   #+begin_src sh
+     sudo apt-get install global
+   #+end_src
 
-#+BEGIN_SRC sh
-  tar xvf global-6.4.tar.gz
-  cd global-6.4
-  ./configure
-  make
-  sudo make install
-#+END_SRC
+*** Install on OSX using Homebrew
 
-To be able to use =ctags= and other backends, you need to copy the sample
-=gtags.conf= either to =/etc/gtags.conf= or =$HOME/.globalrc=. For example:
+    #+begin_src sh options
+      brew install global --with-pygments --with-ctags
+    #+end_src
 
-#+begin_src sh
-  cp gtags.conf ~/.globalrc
-#+end_src
+*** Install on *nix from source
 
-To use this configuration layer, add it to your =~/.spacemacs=. You will need to
-add =gtags= to the existing =dotspacemacs-configuration-layers= list in this
-file.
+**** Install recommended dependencies
 
-* Usage
+    To take full advantage of global you should install 2 extra packages in
+    addition to global: pygments and ctags (exuberant). You can do this using
+    your normal OS package manager, e.g., on Ubuntu
+
+    #+BEGIN_SRC sh
+      sudo apt-get install exuberant-ctags python-pygments
+    #+END_SRC
+
+    or e.g., Archlinux:
+
+    #+BEGIN_SRC sh
+      sudo pacman -S ctags python-pygments
+    #+END_SRC
+
+**** Install with recommended features
+
+    Download the latest tar.gz archive, then run these commands:
 
-Before using the helm-gtags, remember to create a GTAGS database by the following methods:
+    #+BEGIN_SRC sh
+      tar xvf global-6.5.3.tar.gz
+      cd global-6.5.3
+      ./configure --with-exuberant-ctags=/usr/bin/ctags
+      make
+      sudo make install
+    #+END_SRC
 
-- From within Emacs, runs the command =helm-gtags-create-tags=, which is bound
-  to ~SPC m g c~. If the language is not directly supported by GNU Global, you
-  can choose =ctags= or =pygment= as a backend to generate tag database.
+**** Configure your environment to use pygments and ctags
 
-- From inside terminal, runs gtags at your project root in terminal:
+    To be able to use =pygments= and =ctags=, you need to copy the sample
+    =gtags.conf= either to =/etc/gtags.conf= or =$HOME/.globalrc=. For example:
```

Comment:
> In another comment you mentioned AUR, at least for me the AUR package installed gtags.conf in /usr/share/gtags/gtags.conf. In fact my .globalrc is simply a symlink to this file on my archlinux machines. 

That's good to know. It might be worth mentioning in the docs here.